apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -2012,11 +1971,7 @@ machines:
 
 func (s *bundleSuite) TestMixedSeriesNoDefaultSeries(c *gc.C) {
 	s.st.model = description.NewModel(description.ModelArgs{Owner: names.NewUserTag("magic"),
-		Config: map[string]interface{}{
-			"name":           "awesome",
-			"uuid":           "some-uuid",
-			"default-series": "bionic",
-		},
+		Config:      coretesting.FakeConfig(),
```

Comment:
maybe we should still test what happens when `default-series` is in the config to make sure it's not taken into account?

Reply:
This was a bit of a weird one, since the test is called `TestMixedSeriesNoDefaultSeries`

Reply:
Hence why I changed the test a little. I suspect it was included as a mistake

Reply:
If a default base is included, it's identical to the above test `TestMixedSeries` as well

Reply:
That is what the test immediately above this one does

Reply:
NOTE: I have misunderstood the intention behind this test. Fixed in a future PR
https://github.com/juju/juju/pull/16018